libr/fs/shell.c

```diff
@@ -32,6 +32,7 @@ R_API int r_fs_shell_prompt(RFSShell* shell, RFS* fs, const char* root) {
 	for (;;) {
 		snprintf (prompt, sizeof (prompt), "[%.*s]> ", (int)sizeof (prompt) - 5, path);
 		if (shell) {
+			strcpy (shell->cwd, path);
```

Comment:
Better use char*

Reply:
so , what if path is longer than the previous strdup() into cwd? better free and strdup again

Reply:
No matter what I try, what I strdup into shell->cwd doesn't appear in *cwd inside cmd_mount.c

Reply:
Nevermind, I got it. Going to send the PR soon.